projects/frontend/data-pipelines/gui/projects/ui/src/app/app.module.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
+import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+import { AuthConfig, OAuthModule, OAuthStorage } from 'angular-oauth2-oidc';
+
+import { TimeagoModule } from 'ngx-timeago';
+import { LottieModule } from 'ngx-lottie';
+
+import { ApolloModule } from 'apollo-angular';
+
+import { ClarityModule } from '@clr/angular';
+
+import { VmwComponentsModule } from '@vdk/shared';
+
+import { TaurusSharedCoreModule, TaurusSharedFeaturesModule, TaurusSharedNgRxModule } from '@vdk/shared';
+
+import { DataPipelinesModule } from '@vdk/data-pipelines';
+
+import { AppComponent } from './app.component';
+import { AppRouting } from './app.routing';
+import { authCodeFlowConfig } from './auth';
+import { AuthorizationInterceptor } from './http.interceptor';
+import { GettingStartedComponent } from './getting-started/getting-started.component';
+
+// eslint-disable-next-line prefer-arrow/prefer-arrow-functions
+export function lottiePlayerLoader() {
+    return import('lottie-web');
+}
+
+@NgModule({
+    declarations: [AppComponent, GettingStartedComponent],
+    imports: [
+        AppRouting,
+        BrowserModule,
+        ClarityModule,
+        BrowserAnimationsModule,
+        ApolloModule,
+        TaurusSharedCoreModule.forRoot(),
+        TaurusSharedFeaturesModule.forRoot(),
+        TaurusSharedNgRxModule.forRootWithDevtools(),
+        TimeagoModule.forRoot(),
+        LottieModule.forRoot({ player: lottiePlayerLoader }),
+        VmwComponentsModule.forRoot(),
+        DataPipelinesModule.forRoot({
+            defaultOwnerTeamName: 'taurus',
+            manageConfig: {
+                allowKeyTabDownloads: true
+            },
+            exploreConfig: {
+                showTeamsColumn: true
+            },
+            healthStatusUrl: '/explore/data-jobs?search={0}',
+            showExecutionsPage: true,
+            showLineagePage: false
+        }),
+        HttpClientModule,
+        OAuthModule.forRoot({
+            resourceServer: {
+                allowedUrls: [
+                    'https://console-stg.cloud.vmware.com/',
```

Comment:
I believe the 'prod' related URLs are not needed(as we will stick to the staging ones for the moment). 

So, https://console.cloud.vmware.com/ and https://gaz.csp-vidm-prod.com/, should be removed for the timebeing.

Reply:
https://console.cloud.vmware.com/ is already mentioned https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml#L339
made a change to simplify the allowedUrls matching STG needed only;